django/urls/resolvers.py

```diff
@@ -160,41 +163,48 @@ def _route_to_regex(route):
     return ''.join(parts), converters
 
 
-class PathURLMixin:
+class RoutePattern:
+    def __init__(self, route):
+        self.regex = route
+
     def compile(self):
         # TODO: Wrap in `try/except:raise ImproperlyConfigured`
-        regex, converters = _route_to_regex(str(self.pattern))
+        regex, converters = _route_to_regex(str(self.regex))
         if isinstance(self, URLPattern):
```

Comment:
This was originally already hacky, but now also broken.

The purpose of this if was to check if it was a final pattern (view) or not (an include). If it was final, we don't want to allow trailing text.